Jenkinsfile

```diff
@@ -365,7 +371,7 @@ pipeline {
           }
           stage('Playstore') {
             when {
-              expression { env.trackName != 'None' && env.flavor != 'Dev' && env.CHANGE_ID == null }
+              expression { env.trackName != 'None' && env.flavor == 'Beta' && env.CHANGE_ID == null }
```

Comment:
When pushing, jerkins tried to publish to app store even if I was not pushing to main